prime-dotnet-webapi/Services/interfaces/IEnrolleePaperSubmissionService.cs

```diff
@@ -25,5 +25,11 @@ public interface IEnrolleePaperSubmissionService
         Task FinalizeSubmissionAsync(int enrolleeId);
         Task AddEnrolleeAdjudicationDocumentsAsync(int enrolleeId, int adminId, IEnumerable<PaperEnrolleeDocumentViewModel> documents);
         Task<IEnumerable<EnrolleeAdjudicationDocument>> GetEnrolleeAdjudicationDocumentsAsync(int enrolleeId);
+        Task<bool> PotentialReturneeExistsAsync(DateTime dateOfBirth);
+        Task<IEnumerable<Enrollee>> GetPotentialPaperEnrolleeReturneesAsync(DateTime dateOfBirth);
+        Task<bool> LinkEnrolmentToPaperEnrolmentAsync(int enrolmentId, int PaperEnrolmentId, bool isConfirmed = false);
```

Comment:
parameter names shoudl be lower case, so `PaperEnromentId` => `paperEnrolmentId`